npm-react/addons.js

```diff
@@ -1,8 +1,8 @@
 var warning = require('./lib/warning');
 warning(
   false,
-  'require("react/addons") is deprecated. ' +
-  'Access using require("react/addons/{addon}") instead.'
+  'require(\'react/addons\') is deprecated. ' +
```

Comment:
I guess let's keep it as single quotes in the require but use double quotes for the outer string so we don't need the escapes.